### ISSUE TITLE
Support for `pnpm`

### ISSUE THREAD

**Issue by mubaidr** [userquin/vuetify-nuxt-module#47]
When using this module with `pnpm`, following error occurs: 

```bash
> nuxt-module-build --stub && nuxi prepare playground
│ 'nuxt-module-build' is not recognized as an internal or external command,
│ operable program or batch file.
│  ELIFECYCLE  Command failed with exit code 1.
│  WARN   Local package.json exists, but node_modules missing, did you mean to install?
```

**Comment by userquin** [comment]
have you installed dependencies?

**Comment by mubaidr** [comment]
Yes, added module to `nuxt` and `package.json` then ran `pnpm i` and this error pops up.

**Comment by userquin** [comment]
I have no idea how are you installing this module, but it is working in StackBlitz: https://stackblitz.com/edit/nuxt-starter-gd81vx?file=package.json

**Comment by mubaidr** [comment]
Closing as working on web container, must be issue on local system. Sorry for bothering you! 

P.S thanks for awesome work!

**Comment by userquin** [comment]
it is weird, it is running the postinstall script of the module, I'll fix it in the next patch (0.4.6) in a few minutes...

**Comment by mubaidr** [comment]
> it is weird, it is running the postinstall script of the module, I'll fix it in the next patch (0.4.6) in a few minutes...

But more weird it worked in stackblitz.

Thanks
